trace_decoder/src/decoding.rs

```diff
@@ -149,27 +154,19 @@ pub enum TraceParsingErrorReason {
     /// Failure due to trying to access or delete a storage trie missing
     /// from the base trie.
     #[error("Missing account storage trie in base trie when constructing subset partial trie for txn (account: {0:x})")]
-    MissingAccountStorageTrie(HashedAccountAddr),
-
-    /// Failure due to trying to access a non-existent key in the trie.
-    #[error("Tried accessing a non-existent key ({1:x}) in the {0} trie (root hash: {2:x})")]
-    NonExistentTrieEntry(TrieType, Nibbles, TrieRootHash),
+    MissingAccountStorageTrie(H256),
 
     /// Failure due to missing keys when creating a sub-partial trie.
     #[error("Missing key {0:x} when creating sub-partial tries (Trie type: {1})")]
     MissingKeysCreatingSubPartialTrie(Nibbles, TrieType),
 
     /// Failure due to trying to withdraw from a missing account
     #[error("No account present at {0:x} (hashed: {1:x}) to withdraw {2} Gwei from!")]
-    MissingWithdrawalAccount(Address, HashedAccountAddr, U256),
+    MissingWithdrawalAccount(Address, H256, U256),
```

Comment:
Yeah this is probably a bit subjective, but I agree with Robin here in that I think using aliases to make it clear what the data actually is here makes signatures more readable.